src/utils/storage/index.ts

```diff
@@ -17,15 +17,17 @@ enum StorageKeysEnum {
   ProfileData = 'profiledata',
   OtherProfileData = 'otherprofiledata',
   LastPromptNotification = 'lastPromptNotification',
-  LastSelectedMenu = 'lastSelectedMenu'
+  LastSelectedMenu = 'lastSelectedMenu',
+  BlockedStatus = 'blockedStatus',
+  BlockingStatus = 'blockingStatus'
 }
 
 interface IStorage {
-  get: () => string;
+  get: () => string | undefined;
   set: (value: string) => void;
   clear: () => void;
   setForKey: (key: string, value: string) => void;
-  getForKey: (key: string) => string;
+  getForKey: (key: string) => string | undefined;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The return type of `get` and `getForKey` methods in the `IStorage` interface has been changed to include `undefined`. This change is good as it makes the code more robust by explicitly handling the case when the storage does not contain a value for a given key. However, please ensure that all parts of the code that use these methods are updated to handle the `undefined` return type.



<!-- This is an auto-generated comment by OSS CodeRabbit -->